tests/trainer/logging_/test_eval_loop_logging.py

```diff
@@ -590,3 +590,29 @@ def get_metrics_at_idx(idx):
         "test_loss",
     }
     assert set(results[0]) == {"test_loss"}
+
+
+def test_logging_dict_on_validation_step(tmpdir):
+    class TestModel(BoringModel):
+        def validation_step(self, batch, batch_idx):
+            loss = super().validation_step(batch, batch_idx)
+            loss = loss["x"]
+            metrics = {
+                "loss": loss,
+                "loss_1": loss,
+            }
+            self.log("val_metrics", metrics)
```

Comment:
You can repro with dummy values


```suggestion
            self.log("val_metrics", {"a": 1, "b": 2})
```

Reply:
Done !